src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -558,6 +560,9 @@ public async Task<HttpResponseMessage> SendAsyncCore(HttpRequestMessage request,
                     sendRequestContentTask = null;
                 }
 
+                // Now we are sure that the request was fully sent.
+                if (NetEventSource.IsEnabled) Trace("Request is fully sent.");
```

Comment:
Nit: "Request is fully sent" => "Request has been fully sent." or just the simple "Request sent."